backend/src/test/java/wooteco/retrospective/presentation/member/MemberControllerTest.java

```diff
@@ -89,4 +94,41 @@ void loginMemberFailWithNullName() throws Exception {
                 .content(objectMapper.writeValueAsString(request)))
                 .andExpect(status().isBadRequest());
     }
+
+    @DisplayName("멤버를 찾는다 - 정상")
+    @Test
+    void findMember() throws Exception {
+        Member member = new Member(1L, "pika");
+
+        when(memberService.findMemberByToken(any(String.class)))
+                .thenReturn(member);
+        when(jwtTokenProvider.validateToken(any(String.class)))
+                .thenReturn(true);
```

Comment:
```suggestion
        when(jwtTokenProvider.validateToken(any(String.class)))
                .thenReturn(true);
        when(memberService.findMemberByToken(any(String.class)))
                .thenReturn(member);
```

취향 차이? 일 수 있는데, validateToken() 먼저 하고 findMemberByToken() 하는 건 어떨까요?

Reply:
넵! 반영하겠습니다!